com.inochi2d.inochi-session.yml

```diff
@@ -0,0 +1,67 @@
+id: com.inochi2d.inochi-session
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.llvm14
+command: inochi-session
+finish-args:
+  - --device=all          # OpenGL rendering, webcams (may be required? not sure)
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network       # Needed to communicate with external trackers
+  - --socket=wayland
+
+modules:
+  - shared-modules/SDL2/SDL2-with-libdecor.json
+
+  # --- Inochi Session ---
+  - name: Inochi-Session
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/ldc/bin
+    build-commands:
+      - chmod +x ./gentl.sh; ./gentl.sh
+      - dub add-path .flatpak-dub; dub build --skip-registry=all --config=meta; dub
+        build --skip-registry=all --build=release --config=linux-full
+      - install -m 755 ./out/inochi-session /app/bin/inochi-session
+      - install -m 755 ./out/cimgui.so /app/bin/cimgui.so
+      - dub clean --all-packages 
+      # Uncomment when session translations are actually a thing. 
+      #- install -Dm644 ./out/*.mo /app/bin
+
+    sources:
+      - dub-add-local-sources.json
+
+      - type: git
+        url: https://github.com/Inochi2D/inochi-session.git
+        commit: cc0e3a47fbc9b4bb149a917fde0570be6931c5cb
+        tag: v0.8.4
+        disable-shallow-clone: true
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+          is-main-source: true
+
+  - name: Metadata
```

Comment:
this should be with the main modul `inochi-session`

Reply:
Merged :+1: